backend/app/rest/api/rest_private.go

```diff
@@ -236,9 +236,21 @@ func (s *private) updateCommentCtrl(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, res)
 }
 
-// GET /user?site=siteID - returns user info
+// GET /user?site=siteID&unauthorised200=false - returns user info, with unauthorised200=true returns 200 with error message
 func (s *private) userInfoCtrl(w http.ResponseWriter, r *http.Request) {
-	user := rest.MustGetUserInfo(r)
+	user, err := rest.GetUserInfo(r)
+	if err != nil {
+		if r.URL.Query().Get("unauthorised200") == "true" {
```

Comment:
Is the unauthorised200 parameter necessary? It looks like the frontend code will already accept a 200 response with `null` as the JSON response.

Please correct me if I'm wrong, I'm new here, but I think if `null` can be returned here, it looks like no frontend changes would be needed.